docs/2023/Sunbird ED/Frontend And Deep Linking copy/updates/2023-07-07.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Week 1
+author: Om Tanmaya pati
+---
+
+## Milestones
+#### ✨ Milestone 1: Complete Understanding of Sunbird Ed
+In Milestone 1, the focus is on achieving a comprehensive understanding of the Sunbird Ed platform. This involves studying its features, functionalities, and overall architecture.
+#### 🔍 Milestone 2: Understanding Sunbird Ed Code Base
+During Milestone 2, the objective is to gain a deep understanding of the Sunbird Ed code base. This includes analyzing the source code, modules, and how different components interact with each other.
+#### 🏗️ Milestone 3: Setting up Sunbird Ed Locally
+Milestone 3 involves the complete setup of Sunbird Ed on the local host. This includes configuring the development environment, database, and dependencies to ensure a functional local instance of Sunbird Ed.
+
+## Screenshots / Videos 
+![Alt text](<WhatsApp Image 2023-07-11 at 21.03.15.jpg>)
```

Comment:
please change the name of image to proper one

Reply:
Ok